src/pages-and-resources/teams/TeamSetEditor.jsx

```diff
@@ -0,0 +1,193 @@
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+
+import {
+  Button, Collapsible, Form, TransitionReplace,
+} from '@edx/paragon';
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import messages from './messages';
+
+/**
+ * Team sizes enum
+ * @enum
+ * @type {{MIN: number, MAX: number, DEFAULT: number}}
+ */
+export const TeamSizes = {
+  DEFAULT: 50,
+  MIN: 5,
+  MAX: 500,
```

Comment:
Shouldn't these be configurable from the deployment section as well? Something like reading from the environment. As explained in [12factorapp](https://12factor.net/config).

Reply:
So currently these are not configurable in edx-platform either, so I don't see the point of making them configurable here.  I've separated them out so that they can be made configurable in the future. However, simply making them configurable here without a corresponding backend change won't help much.